DEVELOPMENT.md

```diff
@@ -0,0 +1,72 @@
+# Development
+
+This doc explains how to setup a development environment so you can get started
+[contributing](https://www.knative.dev/contributing/) to `Knative Test Infra`. Also
+take a look at:
```

Comment:
<!--mdfmt-bot-->
Format markdown:
```suggestion
[contributing](https://www.knative.dev/contributing/) to `Knative Test Infra`.
Also take a look at:
```

Reply:
@chaodaiG, please confirm if you plan to follow this suggestion. Either way, the change looks good to me